seatunnel-engine/seatunnel-engine-common/src/main/resources/hazelcast.yaml

```diff
@@ -39,3 +39,10 @@ hazelcast:
     hazelcast.tcp.join.port.try.count: 30
     hazelcast.logging.type: log4j2
     hazelcast.operation.generic.thread.count: 50
+  member-attributes:
+    group:
+      type: string
+      value: platform
+    team:
+      type: string
+      value: team1
```

Comment:
Hi, please do not change this. I believe this feature should not open by default.

Reply:
reverted